keps/sig-node/606-compute-device-assignment/README.md

```diff
@@ -179,60 +218,128 @@ Downgrades here are related to downgrading the plugin
-* **Are there any missing metrics that would be useful to have to improve observability if this feature?** No.
+### Monitoring Requirements
 
+###### How can an operator determine if the feature is in use by workloads?
+
+- Look at the `pod_resources_endpoint_requests_total` metric exposed by the kubelet.
+- Look at hostPath mounts of privileged containers.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [ ] Events
+  - Event Reason: 
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [X] Other (treat as last resort)
+  - Details: check the kubelet metrics `pod_resources_endpoint_*`
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A or refer to Kubelet SLIs.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [X] Metrics
+  - Metric name: `pod_resources_endpoint_requests_total`
+  - Components exposing the metric: kubelet
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+No.
 
 ### Dependencies
 
-* **Does this feature depend on any specific services running in the cluster?** Not aplicable.
+###### Does this feature depend on any specific services running in the cluster?
+
+Not applicable.
 
 ### Scalability
 
-* **Will enabling / using this feature result in any new API calls?** No.
-* **Will enabling / using this feature result in introducing new API types?** No.
-* **Will enabling / using this feature result in any new calls to cloud provider?** No.
-* **Will enabling / using this feature result in increasing size or count of the existing API objects?** No.
-* **Will enabling / using this feature result in increasing time taken by any operations covered by [existing SLIs/SLOs][]?** No. Feature is out of existing any paths in kubelet.
-* **Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?** In 1.18, DDOSing the API can lead to resource exhaustion. It is planned to be addressed as part of G.A.
+###### Will enabling / using this feature result in any new API calls?
+
+No.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No. Feature is out of existing any paths in kubelet.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+In 1.18, DDOSing the API can lead to resource exhaustion. It is planned to be addressed as part of G.A.
```

Comment:
How is this being mitigated in the graduation to GA?

Reply:
this was not addressed indeed yet. I'm looking at options at gRPC level.

Reply:
Thank you for acknowledging this risk here.  Please be sure the sig considers this information when making the decision about promoting to GA.

Reply:
@ffromani let's add this into the GA graduation criteria than.

Is this DDOS worse than what can be achieved by querying other endpoints? If not (would be my quess), it may be deserving it's own KEP.

Reply:
This should be addressed and GA blocker.

Reply:
thanks @dchen1107 @SergeyKanzhelev , I'll prepare a followup KEP update to explicitely mention this requisite and make sure it's a GA blocker.